lib/utils/constants.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+
+abstract class Constants{
+  static const Color avatarShadowColor = Colors.black;
+  static const Color personNameTextColor = Colors.white;
+  static const Color contactInfoTextColor = Colors.white;
+  static const Color positionTextColor = Colors.black54;
+  static const Color phoneIconColor = Colors.green;
+  static const Color emailIconColor = Colors.redAccent;
+  static const Color cardsBackgroundColor = Colors.black12;
+  static const double personNameFontSize = 45.0;
+  static const double phoneNumberFontSize = 20.0;
+  static const double positionFontSize = 20.0;
+  static const double personNameAndPositionSizeBoxHeight = 10.0;
+  static const double avatarAndPersonNameSizeBoxHeight = 45.0;
+  static const double phoneAndEmailSizeBoxHeight = 10.0;
+  static const double avatarRadius = 100.00;
+  static const double letterSpacing = 1.00;
+  static const double emailFontSize = 20.0;
+  static const double avatarBlurShadow = 35.0;
+
+}
```

Comment:
You missed an extra line that is need it by convention. Use CTRL+ALT+L to solved it.

Reply:
fixed